src/utils/config.ts

```diff
@@ -37,6 +37,19 @@ export const getCharityAppURL = (): string => {
   }
 }
 
+export const getBlogLocation = (): string => {
+  switch (process.env.GATSBY_RELEASE_ENV) {
+    case 'dev':
+      return 'https://blog.payid.org/'
+    case 'stage':
+      return 'https://blog.payid.org'
+    case 'prod':
+      return 'https://blog.payid.org/'
+    default:
+      return 'https://blog.payid.org/'
+  }
+}
+
```

Comment:
If these are all going to be the same we should just hard code this in the nav

Reply:
Thought about hardcoding but it's possible to have 'dev' for local.  Still remove them for now?

Reply:
Hardcoded for now and fix for spacing